packages/react-server-dom-webpack/flight-hooks.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
You'll need to add this to be built as a bundle too.

We'll also need a `react-server-dom-relay` one for FB.